components/mpas-ocean/src/Registry.xml

```diff
@@ -2118,6 +2185,12 @@
 		<var name="ssh" type="real" dimensions="nCells Time" units="m"
 			 description="sea surface height"
 		/>
+                <var name="vertVelocityNonhydro" type="real" dimensions="nVertLevelsP1 nCells Time" units="m s^{-1}"
+			 description="prognostic vertical velocity, only used when config_enable_nonhydrostatic_mode = .true."
+    		/>
+                <var name="nonhydrostaticPressure" type="real" dimensions="nVertLevels nCells Time" units="N m^{-2}"
+			 description="nonhydrostatic pressure, set to zero if config_enable_nonhydrostatic_mode = .false."
+                />
```

Comment:
Can these variables be put in the `nonhydrostaticPKG` package?  If they aren't used at all when the `nonhydrostaticPKG` is disabled, this should be possible.  If not, this will add to the memory used in hydrostatic mode.